### PR TITLE
Use sealed class for signs with numbers and without it.

### PR DESCRIPTION
**Pull request by yunikkk**

### CODE REVIEW

File: app/src/main/java/com/mapbox/vision/examples/models/UiSign.kt [hunk 1]
```diff
@@ -2,58 +2,63 @@ package com.mapbox.vision.examples.models
...
-                    throw IllegalStateException("Illegal sign! ${signClassification.sign}")
-                }
+
+                check(signTypeIndex < signTypeValues.size) { "Illegal sign! ${signClassification.sign}" }
 
                 val signType = signTypeValues[signTypeIndex]
 
-                val signNumber = if (signTypeIndex > SignType.SpeedLimitRamp.ordinal) {
-                    SignNumber.Unknown
+                if (signType.hasNumber) {
```
Comment (yunikkk) [inline app/src/main/java/com/mapbox/vision/examples/models/UiSign.kt]: Previously signs from the top of enum were treated as those with numbers. New solution is not perfectly elegant either but still seems much better to me. If someone has better ideas lets discuss

File: app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt [hunk 2]
```diff
@@ -48,7 +48,9 @@ import com.mapbox.vision.performance.ModelPerformanceRate
 import com.mapbox.vision.safety.VisionSafetyManager
 import com.mapbox.vision.safety.core.VisionSafetyListener
 import com.mapbox.vision.safety.core.models.CollisionDangerLevel
-import com.mapbox.vision.safety.core.models.CollisionDangerLevel.*
```
Comment (korshaknn) [inline app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt]: ktlint

File: app/src/main/java/com/mapbox/vision/examples/models/UiSign.kt (hunk 1 above)
Comment (RingerJK) [reply app/src/main/java/com/mapbox/vision/examples/models/UiSign.kt]: it looks like in future we'll separate numbers by region. In different counties the same signs could have or have no numbers

Comment (yunikkk) [reply app/src/main/java/com/mapbox/vision/examples/models/UiSign.kt]: Mmm, do you have some specific examples in mind? 
I think signs will be different then, there is no reason to name signs with the same name if one contains number and another does not.

File: app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt (hunk 2 above)
Comment (yunikkk) [reply app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt]: \+

Comment (yunikkk) [reply app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt]: Well, turns out `ktlint` isn't applied in Teaser yet. I've submitted an issue to apply it https://app.zenhub.com/workspaces/vision-sdk-5ca52e6c027e3e7234ac7512/issues/mapbox/mapbox-vision/1700

**Pull request merged**